packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/ai-agent-action/components/AIChatTab.tsx

```diff
@@ -0,0 +1,261 @@
+import { TextArea } from '@/ui/input/components/TextArea';
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import React from 'react';
+import { Avatar, IconDotsVertical, IconSparkles } from 'twenty-ui/display';
+
+import { LightCopyIconButton } from '@/object-record/record-field/components/LightCopyIconButton';
+import { ScrollWrapper } from '@/ui/utilities/scroll/components/ScrollWrapper';
+import { AgentChatMessageRole } from '@/workflow/workflow-steps/workflow-actions/ai-agent-action/constants/agent-chat-message-role';
+import { t } from '@lingui/core/macro';
+import { Button } from 'twenty-ui/input';
+import { beautifyPastDateRelativeToNow } from '~/utils/date-utils';
+import { useAgentChat } from '../hooks/useAgentChat';
+import { AIChatSkeletonLoader } from './AIChatSkeletonLoader';
+
+const StyledContainer = styled.div`
+  background: ${({ theme }) => theme.background.primary};
+  height: calc(100% - 154px);
+`;
+
+const StyledEmptyState = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  gap: ${({ theme }) => theme.spacing(2)};
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+`;
+
+const StyledSparkleIcon = styled.div`
+  align-items: center;
+  background: ${({ theme }) => theme.background.transparent.blue};
+  border-radius: ${({ theme }) => theme.border.radius.sm};
+  display: flex;
+  height: 40px;
+  justify-content: center;
+  margin-bottom: ${({ theme }) => theme.spacing(2)};
+  width: 40px;
+`;
+
+const StyledTitle = styled.div`
+  font-size: ${({ theme }) => theme.font.size.lg};
+  font-weight: 600;
+`;
+
+const StyledDescription = styled.div`
+  color: ${({ theme }) => theme.font.color.secondary};
+  text-align: center;
+  max-width: 410px;
```

Comment:
Check your CSS for hardcoded values which are often code smell and means things can be done differently. The design should react well to different panel size for example